src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -36,6 +37,36 @@ export type SurvivalPlotFilters = {
     y: [number, number];
 };
 
+// TODO should remove this if we want to be generic
+export const survivalClinicalDataVocabulary: { [prefix: string]: string[] } = {
+    OS: ['DECEASED'],
+    PFS: ['CENSORED'],
+    DFS: ['Recurred/Progressed', 'Recurred'],
+    DSS: ['DEAD WITH TUMOR'],
+    EFS: ['Censored'],
+};
+
+// TODO should remove this if we want to be generic
+export const survivalPlotTooltipxLabelWithEvent: {
+    [prefix: string]: string;
+} = {
+    // TODO text for PFS DSS EFS?
+    OS: 'Time of death',
+    PFS: 'Time of death',
+    DFS: 'Time of Relapse',
```

Comment:
relapse should not be capitalized.  Alternatively, "death" should be capitalized